<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_remove_user_from_project</class>
 <widget class="QDialog" name="Dialog_remove_user_from_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>140</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_project">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Name of project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>my project</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
